lib/rules/block-indentation.js

```diff
@@ -477,7 +494,7 @@ export default class BlockIndentation extends Rule {
 
           fixedNode = this.fixLine(
             lines,
-            childStartLine - 1,
+            childStartLine - (path ? 1 : fixedNode.loc.start.line),
```

Comment:
This actually seemed to be an existing bug. When I ran the autofixer with `master` on our own embedded templates, then I got `TypeError: Cannot read property 'startsWith' of undefined` because the `childStartLine` was wrong